packages/framework/src/hooks/useScreenData.ts

```diff
@@ -49,11 +51,14 @@ export const useScreenData = (): {
     [data, setDataAtom],
   );
 
-  return {
-    apis,
-    sockets,
-    data,
-    setData,
-    mockResponses,
-  };
+  return useMemo(
```

Comment:
Why is this needed again?

Reply:
I think for this hook, unlike the action hooks, since it is used so rarely that we don't need to memoize the return object but fix the usages instead.